client/maps/map-image.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import ImageIcon from '../icons/material/baseline-image-24px.svg'
+
+import { grey800 } from '../styles/colors'
+import { Subheading } from '../styles/typography'
+
+const ImgElement = styled.img`
+  display: block;
```

Comment:
I'm not sure these styles really make sense for the generally usable component, it'd feel more appropriate for the particular usage in the map browser or whatever

Reply:
`display: block` is needed in lobbies actually, but I get your point. Would you say that the `img` element should have no styling in general use case?

Reply:
That particular style was the least concerning of them, I was more talking about the following two. I think generally I'd try to avoid specifying layout-type things in the styles for base UI components, as it makes them harder/weirder to use in specific UIs. For example, it'd be weird to set the width of a button to 100%, as it'd then require you to put it in a container with a particular size to make it smaller (or override the style with styled-components, I suppose, but same idea). Not applicable *all* the time, but I think you at least need a really good reason to include stuff like that at the root level of a UI component.